CHANGELOG.md

```diff
@@ -4,6 +4,17 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [2.1.0] - 03/20/2023
+
+### New:
```

Comment:
When we were working on the Media Insights release, the `viperlight - publisher check` barked at us for not following [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) to the t.

i.e. adding `:` after each `New`, `Changed`, etc

Reply:
Yep, the post-scan stage in the pipeline scans this file and has a lot to say about it.